--- origsrc/kstars-16.08.3/kstars/htmesh/SkipListElement.h	2016-10-21 10:59:15.000000000 -0500
+++ src/kstars-16.08.3/kstars/htmesh/SkipListElement.h	2016-11-14 02:00:32.291917400 -0600
@@ -22,7 +22,7 @@
 #  if defined(__FreeBSD__) || defined(__NetBSD__) || defined(__OpenBSD__) || defined(__DragonFly__) || ( defined(__SUNPRO_CC) && defined(__sun) )  || defined(__APPLE__)
 #    define KEY_MAX LLONG_MAX
 #  else
-#    define KEY_MAX LONG_LONG_MAX
+#    define KEY_MAX __LONG_LONG_MAX__
 #  endif
 #endif
 
