--- origsrc/kstars-17.08.3/kstars/htmesh/CMakeLists.txt	2017-10-04 23:42:17.000000000 -0500
+++ src/kstars-17.08.3/kstars/htmesh/CMakeLists.txt	2017-11-13 15:33:58.503638000 -0600
@@ -40,10 +40,6 @@ set_target_properties(htmesh PROPERTIES
     VERSION 1.0.0
     SOVERSION 1)
 
-if (NOT ANDROID)
-    install(TARGETS htmesh ${KDE_INSTALL_TARGETS_DEFAULT_ARGS} )
-endif ()
-
 # If you wish to compile the HTMesh perl wrapper, uncomment this, rebuild and copy the library into /usr/lib/, because we will use it as a shared object. See README in the perl wrapper directory (kstars/kstars/data/tools/HTMesh-*) for more details.
 #set_property(TARGET htmesh PROPERTY POSITION_INDEPENDENT_CODE YES)
 
